cmd/cmrel/cmd/generate_tests.go

```diff
@@ -0,0 +1,231 @@
+/*
```

Comment:
I had to double take because of the file suffix `_tests.go`, which can be confused with `_test.go`. 

Can we change it to `generate_prow.go`, and make the corresponding changes to the command help output comments/function names to talk about _prow config_?

Reply:
Good suggestion, yeah, thanks